packages/create-neutrino-project/commands/init/generator/writing.js

```diff
@@ -0,0 +1,142 @@
+const { ensureDirSync, readJsonSync, writeJsonSync } = require('fs-extra');
+const path = require('path');
+const stringify = require('javascript-stringify');
+const utils = require('./utils');
+
+const PROJECTS = utils.PROJECTS;
+const LIBRARY = utils.LIBRARY;
+const LINTERS = utils.LINTERS;
+
+function makeRcFile(data) {
+  let config = { use: [] };
+
+  config.use.push(data.projectType);
+
+  switch (data.linter) {
+    case LINTERS.AIRBNB:
+      config.use.push(
+        data.projectType === PROJECTS.WEB ? LIBRARY.NEUTRINO_PRESET_AIRBNB_BASE : LIBRARY.NEUTRINO_PRESET_AIRBNB
```

Comment:
I think this should probably be:

```js
data.projectType === PROJECTS.REACT ? LIBRARY.NEUTRINO_PRESET_AIRBNB : LIBRARY.NEUTRINO_PRESET_AIRBNB_BASE
```

since we may want to use Airbnb-base on more than just web projects.